protos/foundation/protos/doc/entity.proto

```diff
@@ -2,11 +2,13 @@ syntax = "proto2";
 
 package doc;
 
+import "foundation/protos/geometry.proto";
 import "foundation/protos/ocr/types.proto";
 
+/* Union type holds one of these Entity variants. */
 message Entity {
 	oneof payload {
-		OcrWord ocr_word = 1;
+		Word word = 1;
```

Comment:
I would use "DollarValue", "GBPValue", etc., instead of "currency". What is the trade-off from the user's perspective?

Reply:
I would use `PersonName` instead of name (assuming this is what `Name` refers to). There is too much ambiguity with `Name`.

@vontell ?

Reply:
Remind me what the point is of having a `Token` type?

Reply:
What about either `Decimal`, `Integer`, and `Number` all being distinct, or having only `Number`?

My vote would maybe go to the latter. In my experience periods get dropped a lot in OCR so differentiating among these is not gonna be reliable, which will be surprising to users.

Reply:
What is the difference between a `Phrase` and a `Cluster`?

Reply:
What is the difference between `Paragraph` and `Cluster`, what is the difference between `Line` and `Phrase`?

Reply:
The difference between `Paragraph` and `Cluster`, as I saw it when I made the distinction, was that many OCR engines provide `paragraph`s, but `cluster`s could be built by many alternative means, including potential model-service uses where we'd like to associate a group of words with a label (e.g. NER). This may not be a useful distinction however. It may make more sense to drop `Cluster` and make use of a tuple-like struct `LabeledEntity`.

Reply:
`Line` vs `Phrase` is similar (OCR provides `line`s), but given the discussion about `Token`, I think we could just drop `Phrase` and `Token` both.